### ISSUE TITLE
Possible to display namespace of a property/parameter in output?

### ISSUE THREAD

**Issue by RoRoche** [bafolts/tplant#54]
Hi,

Thank you for this amazing tool.

When a property/parameter is of a type belonging to a namespace, is there a way to display this namespace in the generated PlantUML diagram?

**Comment by bafolts** [comment]
Most likely. Can you provide expected input and output?

**Comment by RoRoche** [comment]
Input:

```typescript
namespace Login {
  class Request {
    // code
  }
}

// anywhere:

class LoginComponent {
  public request: Login.Request;
}
```

Expected plantuml:

```plantuml
@startuml
namespace Login {
  class Request {
    // ...
  }
}

class LoginComponent {
  +request: Login.Request
}
@enduml
```

**Comment by bafolts** [comment]
The #58 will fix for properties.

**Comment by bafolts** [comment]
The #59 fixed parameters and method return types. Available in 2.3.5
